sdk/typescript/src/types/objects.ts

```diff
@@ -36,6 +36,21 @@ export type SuiMoveObject = {
   has_public_transfer?: boolean;
 };
 
+export type CoinDenominationInfoResponse = {
+  /** Coin type like "0x2::sui::SUI" */
+  coin_type: string;
```

Comment:
```suggestion
  coinType: string;
```

nit: I know it's not very consistent right now (due to RPC response schema), but let's try to use camelCase whenever possible